src/main/java/com/example/fooddelivery/forms/AddressForm.java

```diff
@@ -0,0 +1,31 @@
+package com.example.fooddelivery.forms;
+
+public class AddressForm {
+    private String city;
+    private String street;
+    private String number;
```

Comment:
why don't you use integer or long?

Reply:
For House numbers for example. We can have nr. 4a